Dockerfile

```
@@ -19,6 +21,12 @@ COPY --chown=node . .
 RUN RAZZLE_API_PATH=VOLTO_API_PATH RAZZLE_INTERNAL_API_PATH=VOLTO_INTERNAL_API_PATH yarn build && \
     rm -rf /home/node/.cache
 
+USER root
+RUN apt-get purge ${BUILD_DEPS} -y && \
```

```suggestion
RUN apt-get purge $BUILD_DEPS -y && \
```
Toglierei le parentesi giusto per coerenza con l'altra aggiunta sopra :)